CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to contribute
+
+## How to get started
+
+Before anything else, please install the git hooks that run automatic scripts during each commit and merge to strip the notebooks of superfluous metadata (and avoid merge conflicts). After cloning the repository, run the following command inside it:
+```
+nbdev_install_git_hooks
+```
```

Comment:
Would be nice to elaborate a bit more on the `Dev Cycle`, maybe also add a linter (black?)?

Reply:
Maybe add link to: https://github.com/fastai/nbdev/ to get people started ...

Reply:
yes, this is the standard text from nbdev, but its interesting that they did not include these links :)